src/resources/ClusterRoleBindings/ClusterRoleBindingDescription.js

```diff
@@ -0,0 +1,14 @@
+import { Trans } from 'react-i18next';
+import { Link as ReactSharedLink } from '../../shared/components/Link/Link';
```

Comment:
Adjust this to all of the rest of changed files.
```suggestion
import { Link as ReactSharedLink } from 'shared/components/Link/Link';
```